index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>POC on Java sscript</title>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">  
+</head>
+
+
+<body onload="getobject('person')">
```

Comment:
your site is not protected if people can access your functions from outside of javascript file